desktop/apps/artwork/components/partner_stub/components/inquire_via_phone_modal.coffee

```diff
@@ -0,0 +1,29 @@
+AuthModalView = require '../../../../../components/auth_modal/view.coffee'
```

Comment:
I think this is totally fine, it does seem like both of the implemented methods are overrides (can maybe DRY up and use `super`), so this seems 👍 to me.

I also think allowing the title to be a parameter passed in feels fine as well- seems sort of natural for a modal. If we got into a situation where every use of the modal was passing in more and more options and needing more and more customization we should probably refactor more, but a title seems fine to me.